cmd/cmd.go

```diff
@@ -5,9 +5,11 @@ import (
 	"os"
 )
 
-// Bail handles exitable errors in commands.
+// BailOnError handles exitable errors in commands.
 // TODO: figure out what goes here.
-func Bail(err error) {
-	log.Println(err)
-	os.Exit(1)
+func BailOnError(err error) {
```

Comment:
Nah, not really. I haven't figured out what the behavior on bail should be yet, so I just stuck something in there as a placeholder.

Before we release the CLI we should definitely talk about this!